app/http.php

```diff
@@ -1272,23 +1272,28 @@ function (string $runtimeId, ?string $payload, string $path, string $method, mix
 
             // Execute function
             $executionRequest = $version === 'v4' ? $executeV4 : $executeV2;
-            $executionResponse = \call_user_func($executionRequest);
+            do {
+                $executionResponse = \call_user_func($executionRequest);
+            } while (
```

Comment:
Maybe its just me and mobile UI, but I cant find sleep. Lets ensyre we sleep like other loops, otherwise this could tske entirity of cpu since its coroutine-style http server

Reply:
I've added sleep, I don't there's not a risk with cpu because curl_exec is non blocking, but probably sensible to keep avoid the network io